Makefile

```diff
@@ -44,7 +44,7 @@ node_modules: package.json package-lock.json
 	node skip_integrity_check.js
 
 ifeq ($(CI),false)
-	npm install --force
```

Comment:
🥳
In my personal opinion forking repositories in mattermost is better than force.